test/e2e/cassandra.go

```diff
@@ -15,7 +16,11 @@ import (
 
 // Cassandra runs a test with Cassandra as the backing storage
 func Cassandra(t *testing.T) {
-	ctx := prepare(t)
+	ctx, err := prepare(t)
```

Comment:
Could we call `prepare` in the caller so each test would not have to repeat this block?

Reply:
@pavolloffay Once I finish the other changes I am making for reporting it will be called in the Setup method, only once per suite.  https://github.com/jaegertracing/jaeger-operator/blob/master/test/e2e/all_in_one_test.go#L36-L44

I've run into this bug a few times though, so I wanted to get this done first.